tests_integration/__snapshots__/test_sql.ambr

```diff
@@ -0,0 +1,16 @@
+# serializer version: 1
+# name: test_multi_queries_where_one_of_them_is_failing
+  '''
+  [
+  [
+      {
+          "1": 1
+      }
+  ],
+  [
+      {
+          "2": 2
+      }
+  ]
+  '''
```

Comment:
```
➜ snow sql -c int -q "select 1; select 2; select foo; select 4" --format=json          
[
[
    {
        "1": 1
    }
],
[
    {
        "2": 2
    }
]
An unexpected exception occurred. Use --debug option to see the traceback. Exception message:

000904 (42000): SQL compilation error: error line 1 at position 7
invalid identifier 'FOO'
```

This is the actual output, for some reason the error is redacted when using output from runner. It is present as `result.exception`